### DISCUSSION TITLE
Gifs not really freezing

### DISCUSSION THREAD

**Discussion by GreatZamboni** [AprilSylph/XKit-Rewritten#612] [Help Desk]
I keep having an issue where the frozen gifs are of their original size, while moving ones are sized up or down according to the post and tumblr requirements. Sometimes the original gif is larger than what's included in post, meaning the text under the gifs is covered by the frozen image. Usually the original is smaller and the moving (a tad stretched version) is very visible underneath. I feel it defeats the purpose of the accessibility option.
Edit: [I made a gif to show what I mean.](https://gyazo.com/411f5338e5a297eca54b2cf5a5388e88)

**Comment by marcustyphoon** [comment]
I wonder if this is related to this line from [the most recent post on changes](https://www.tumblr.com/blog/view/changes/684791420492578816):

> We’re ramping a performance improvement on web that should reduce the amount of time you’ll see the dreaded gradient placeholder instead of GIFs. If the GIF is still loading slowly, you’ll see the first frame of the GIF while it loads.

**Reply by GreatZamboni** [reply]
It's not a gradients placeholder though. I [just made a gif to show what I mean](https://gyazo.com/411f5338e5a297eca54b2cf5a5388e88), I also edited the post above.

**Reply by marcustyphoon** [reply]
Oh, sorry, let me clarify: I wonder if the changes mentioned there caused our current code to break in an unexpected way! Maybe the "first frame of the GIF" is added in a way that changes the structure of the page code to one our overlay doesn't expect.

Thanks for the gif, that illustrates what you're seeing nicely.

**Reply by vagrantenchilada** [reply]
I'm having this exact same problem! I toggled the pause gif part of Xkit off and that fixed the problem, though then of course gifs no longer paused at all. Toggling it back on, however, paused the gifs again at the correct size. I attached some screenshots, but the gif made by GreatZamboni captures it best.

![Screenshot (154)](https://user-images.githubusercontent.com/106413564/170802798-25b2bdf5-50b1-47af-a684-0d2c218d0210.png)
![Screenshot (156)](https://user-images.githubusercontent.com/106413564/170802802-3bba9807-318c-4a31-8af5-63123548822c.png)

**Comment by marcustyphoon** [comment]
Aw, issues that mention discussions apparently don't get auto-linked!

This should be fixed by #615; see if it still occurs in the next release!
